tests/app.test.js

```diff
@@ -0,0 +1,93 @@
+import path from 'path';
+import fs from 'fs';
+import React from 'react';
+import { fireEvent, render, wait } from '@testing-library/react'
+import '@testing-library/jest-dom'
```

Comment:
What's the significance of the leading `@`?

Reply:
Good question. My reading is, `jest-dom` is the package, and it's developed under the scope of `@testing-library` project. Mainly that "scoping" has implications for npm. There's no other way to do the import here.

https://stackoverflow.com/questions/36667258/what-is-the-meaning-of-the-at-prefix-on-npm-packages

Reply:
Ah!  That is a really interesting feature, and a bit surprising that it's implemented with some different import syntax.